### PR TITLE
Edit metadata.yaml to update/reorder categories on docs.vf.io

### PR DESCRIPTION
**Pull request by kwm14**
## Done

- Updated metadata.yaml to match work completed on new categorisation on docs.vf.io
- Links remain the same just a reorder and grouping of components
- Design issue found [here](https://github.com/ubuntudesign/vanilla-design/issues/118) which was the outcome from Documentation workshop

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/

## Additonal

- @nottrobin or @WillMoggridge to create/share demo of PR so I can review before going live.

### CODE REVIEW

File: docs/en/metadata.yaml [hunk 1]
```diff
@@ -1,48 +1,72 @@
 navigation:
 
-- title: Settings
+- title: Basics
   children:
 
-  - location: settings/animation-settings.md
```
Comment (WillMoggridge) [inline docs/en/metadata.yaml]: There have been some spaces deleted in the file which changes the indenting and breaks the file.
The file can be validated at https://codebeautify.org/yaml-validator if you want to check yourself.

The child items need two spaces before the `-`, and `title` should line up with `location`

Comment (kwm14) [reply docs/en/metadata.yaml]: @WillMoggridge thanks for reviewing. I will take a look and update based on your feedback, I didn't realise there was a particular layout to this file.

**Comment by WillMoggridge** [comment]
Threw up a quick demo here https://en-inrtdlndnv.now.sh/

The demo isn't perfect but it that is unrelated to this code, the links don't work and need .html appended to them. You can see the menu layout working though.

**Comment by nottrobin** [comment]
Can this be merged?

**Comment by kwm14** [comment]
@nottrobin not yet. Will need Spencer to Design +1 tomorrow once he is back in the office.

**Pull request merged**